### PR TITLE
feat(cli): model eval command support use-docker flag

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
model eval command support use-docker flag
Tested with 4 commands(starwhale==0.3.3.dev1212202632):
1. swcli -vvv model eval --dataset mnist/version/latest mnist/version/latest --image runtime1 --use-docker
2. swcli -vvv model eval --dataset mnist/version/latest . --image runtime1 --use-docker
3. swcli -vvv model eval --dataset mnist/version/latest . --runtime pytorch/version/latest --use-docker
4. swcli -vvv model eval --dataset mnist/version/latest mnist/version/latest --runtime pytorch/version/latest --use-docker

docker file for runtime1, work dir is pytorch runtime code:
```shell
FROM 10.131.0.1:8083/starwhale:0.3.3.dev1212202632
COPY requirements-sw-lock.txt /
RUN python3 -m pip install -r /requirements-sw-lock.txt
RUN python3 -m pip install starwhale==0.3.3.dev1212202632
```

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [x] add necessary doc

### CODE REVIEW

File: client/starwhale/core/model/view.py [hunk 1]
```diff
@@ -11,6 +11,10 @@
 from starwhale.core.runtime.process import Process as RuntimeProcess
 
 from .model import Model, StandaloneModel
+from ...consts.env import SWEnv
```
Comment (goldenxinxing) [inline client/starwhale/core/model/view.py:14]: use from starwhale.const/utils/runtime?

File: client/starwhale/utils/docker.py [hunk 2]
```diff
@@ -149,3 +152,80 @@ def buildx(
...
+    for _ee in (
+        "SW_PYPI_INDEX_URL",
+        "SW_PYPI_EXTRA_INDEX_URL",
+        "SW_PYPI_TRUSTED_HOST",
+    ):
+        if _ee not in _env:
+            continue
+        cmd.extend(["-e", f"{_ee}={_env[_ee]}"])
+
+    sw_cmd = " ".join([item for item in sys.argv[1:] if "use-docker" not in item])
```
[the remark this answers is not in the document]
Comment (anda-ren) [reply client/starwhale/utils/docker.py:228]: yes. however `eval run` could adapt to this function easily

File: client/starwhale/core/model/view.py (hunk 1 above)
Comment (anda-ren) [reply client/starwhale/core/model/view.py]: done

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**